packages/wallets/src/features/cashier/flows/WalletWithdrawal/WalletWithdrawal.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useState } from 'react';
 import { useCountdown } from 'usehooks-ts';
 import { useSettings, useVerifyEmail } from '@deriv/api';
-import { WalletWithdrawalVerification, WalletWithdrawalVerificationSent } from '../../screens';
+import { WithdrawalVerificationModule, WithdrawalVerificationSentModule } from '../../modules';
```

Comment:
Even `WithdrawalVerificationModule` and `WithdrawalVerificationSentModule`, these are screens. What we have in flows should be the modules. Like `WithdrawalFiatModule` and `WithdrawalCryptoModule` with the logic for these screens handled in it.